src/ui/components/UserMenu/UserMenu.jsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
 import { withRouter } from 'react-router';
 
-import Menu from 'material-ui/Menu';
-import MenuItem from 'material-ui/MenuItem';
+import { Menu, MenuItem, Divider } from 'material-ui';
```

Comment:
Habe diese zur Übersicht eingeführt. In der Componente `UserMenuByRole` findest du die einzelnen MenuItem Definitionen jeweils pro Role (TM, QM & Fachjury).